### PR TITLE
[BACKLOG-32839] Fix SaveAs dialog's file tree not refreshing after a Save

### PR DESCRIPTION
**Pull request by davidmsantos90**
@pentaho/millenniumfalcon please review
 - only includes code cleanup

Merge with:
 - https://github.com/pentaho/pentaho-commons-gwt-modules/pull/749

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 5m 34s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl user-console
```
---
:ok_hand: All tests passed!
 
Tests run: **12**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/platform/job/PR-4609/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: user-console/src/main/java/org/pentaho/mantle/client/commands/OpenFileCommand.java [hunk 1]
```diff
@@ -14,29 +14,24 @@
  * See the GNU Lesser General Public License for more details.
  *
  *
- * Copyright (c) 2002-2018 Hitachi Vantara. All rights reserved.
+ * Copyright (c) 2002 - 2019 Hitachi Vantara. All rights reserved.
```
Comment (ricardosilva88) [inline user-console/src/main/java/org/pentaho/mantle/client/commands/OpenFileCommand.java]: should be 2020 :P

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 5m 33s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl user-console
```
---
:ok_hand: All tests passed!
 
Tests run: **12**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/platform/job/PR-4609/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: user-console/src/main/java/org/pentaho/mantle/client/commands/SaveCommand.java [hunk 2]
```diff
@@ -83,30 +81,19 @@ protected void performOperationAsync( boolean feedback, final CommandResultCallb
...
 
-      if ( isSaveAs ) {
-        dialog.setTitle( Messages.getString( "saveAs" ) ); //$NON-NLS-1$
-        dialog.setText( Messages.getString( "saveAs" ) ); //$NON-NLS-1$
-      } else {
-        dialog.setTitle( Messages.getString( "save" ) ); //$NON-NLS-1$
-        dialog.setText( Messages.getString( "save" ) ); //$NON-NLS-1$
-      }
+      dialog.setTitle( modeText );
+      dialog.setText( modeText );
```
Comment (dcleao) [inline user-console/src/main/java/org/pentaho/mantle/client/commands/SaveCommand.java]: Unfortunately, AFAICT, the code is not equivalent. The OK button label will become "Save As", while, originally, it was always "Save"...
Those two text arguments end up in: `PromptDialogBox( String title, String okText, ...)` (see [here](https://github.com/pentaho/pentaho-commons-gwt-modules/blob/dcefb7876203f7ba748766beed8f625d30c9d2d8/widgets/src/main/java/org/pentaho/gwt/widgets/client/dialogs/PromptDialogBox.java#L42)).

The title argument is passed to `setText`, and so your code preserves behavior w.r.t. to this argument.
However, the `okText` is passed directly to the OK button's text/label.

Later, calling `setTitle` affects some kind of tooltip, which the constructor did not set, initially.
While, calling `setText` call only affects the dialog's "caption"...
The OK button will not be changed again.

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 5m 26s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl user-console
```
---
:ok_hand: All tests passed!
 
Tests run: **12**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/platform/job/PR-4609/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by davidmsantos90** [comment]
@dcleao nice catch there, I totally missed that change in behavior. updated the pull request, please review

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 5m 35s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl user-console
```
---
:ok_hand: All tests passed!
 
Tests run: **12**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/platform/job/PR-4609/4/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**